tests/wpt/tests/css/css-tables/table-collapsed-row-or-column-crash.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<div style="display: table; visibility: collapse">
```

Comment:
You can add `<link rel="help" href="https://github.com/servo/servo/issues/37421">` for context.